validator/webui/index.html

```diff
@@ -260,152 +265,199 @@
+        margin: 10px 0;
+        width: auto;
+        text-transform: none;
+        white-space: nowrap;
+        background-image: url('data:image/svg+xml;utf-8,<svg width=\22 14px\22  height=\22 14px\22  viewBox=\22 0 0 14 14\22  version=\22 1.1\22  xmlns=\22 http://www.w3.org/2000/svg\22  xmlns:xlink=\22 http://www.w3.org/1999/xlink\22 ><title>AMP Logo</title><g id=\22 Page-1\22  stroke=\22 none\22  stroke-width=\22 1\22  fill=\22 none\22  fill-rule=\22 evenodd\22 ><g id=\22 AMP-Logo\22  fill=\22 #0379C4\22 ><path d=\22 M8.657817,0.216062 L7.647297,5.589962 L10.085327,5.589962 C10.188787,5.589962 10.256477,5.698322 10.211117,5.791282 L6.230217,13.958532 C6.480467,13.985622 6.734567,13.999972 6.992097,13.999972 C10.853717,13.999972 13.984117,10.869572 13.984117,7.007952 C13.984117,3.720402 11.715067,0.963312 8.657817,0.216062\22  id=\22 Fill-4\22 ></path><path d=\22 M6.425398,8.529948 L3.987158,8.529948 C3.883698,8.529948 3.815938,8.421588 3.861298,8.328628 L7.885808,0.072478 C7.591808,0.034958 7.292138,0.015708 6.987848,0.015848 C3.168228,0.018088 0.028028,3.136308 0.000168,6.955858 C-0.024192,10.303328 2.304078,13.111168 5.429718,13.824608 L6.425398,8.529948 Z\22  id=\22 Fill-1\22 ></path></g></g></svg>');
+        background-repeat: no-repeat;
+        background-size: 40px 40px;
+        background-position: center left;
+      }
+      .ampproject a:link { color: #0379C4; text-decoration: none; }
+      .ampproject a:visited { color: #0379C4; text-decoration: none; }
+      .ampproject a:hover { color: #0379C4; text-decoration: none; }
+      .ampproject a:active { color: #0379C4; text-decoration: none; }
+    </style>
+    <template>
+      <paper-toolbar>
+        <div class="ampproject">
+          <a href="https://www.ampproject.org/">Accelerated Mobile Pages Project Validator</a>
+        </div>
+      </paper-toolbar>
+    </template>
+    <script>
+      Polymer({
+        is: 'styled-toolbar'
+      });
+    </script>
+  </dom-module>
+
   <!-- Below, the main contents of the page, which instantiates the
        custom elements defined above. -->
-  <paper-toolbar>
-      <div class="ampproject">
-        <a href="https://www.ampproject.org/">Accelerated Mobile Pages Project</a>
-      </div>
-  </paper-toolbar>
-
-  <script>
-    function setExample(featureTestName) {
-      var urlToFetch = document.getElementById('urlToFetch');
-      urlToFetch.value =
-        'https://raw.githubusercontent.com/ampproject/amphtml/master/validator/testdata/feature_tests/' + featureTestName;
-      urlToFetch.validate();
-      document.getElementById('validateButton').click();
-    }
-  </script>
-
-  This web UI runs
-  the <a href="https://github.com/ampproject/amphtml/tree/master/validator">AMP
-  Validator</a> to verify that an HTML page conforms to
-  the <a href="https://www.ampproject.org/docs/reference/spec.html">AMP
-  HTML specification</a>. Enter HTML directly, copy/paste, or load by
-  specifying a URL and clicking VALIDATE.
```

Comment:
Minor, not required: Since we're removing the "documentation" of this feature, maybe we should consider adding a "help" page that would describe what it does.